.changeset/curvy-foxes-know.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/toggle': minor
+'@twilio-paste/core': minor
+---
+
+[Toggle] add Toggle package
```

Comment:
I kinda disagree with that even though there's prior art.  Core upgrading majors because we added a package is scary for consumers with no effect. They'll stop taking majors seriously if each new component is major. It should be a minor and either start at 1.0.0 (jumps to 1.1.0) or start at 0.0.0 (jumps to 0.1.0) until the first breaking change in that package which promotes it to 1.0.0

Reply:
looking through changelogs it looks like most of the components start as 0.1.0, but there are definitely some that start at 1.0.0... would there be any reason for that or is it just inconsistency? if not, I think we should document somewhere which one we should stick to

Reply:
So, full clarity here. 

If you release a component as a `0.x.x` is can cause changesets to cascade major updates where you don't want them to. So we should release components as `1.x.x` to prevent that. They are also not alpha.

But releasing a components as `1.x.x` doesn't mean you need to mark core as a major. You can split the change set versions

```
---
'@twilio-paste/toggle': major
'@twilio-paste/core': minor
---
```

So that will bump toggle to v1 and bump core as a minor.

Reply:
You can equally just start the package at `1.0.0` and make it a minor 🤷🏻‍♂️